trunk/research/players/js/srs.sdk.js

```diff
@@ -716,3 +716,28 @@ function SrsRtcFormatSenders(senders, kind) {
     return codecs.join(", ");
 }
 
+function SrsRtcFormatStats(stats, kind) {
```

Comment:
Remove SrsRtcFormatSenders if it is no longer used.

Reply:
Yeap